README.md

```diff
@@ -143,13 +148,50 @@ The image will validate that your SPT version in the serverfiles matches the ima
+
+### What it does
+When the container starts, before it runs the SPT server executable, this will download the provided URLs, extract all the supported file types, and then do the following:
+- Move the `BepInEx/plugins` and `user/mods` to their appropriate locations (effecitvely "installing" them, just like you'd do drag & drop in a local SPT install)
+- Move any bare .dll files to BepInEx/plugins
+- Move any .txt, .md. and .exe files to the root of the mounted directory.
+  - Some mods are or come with .exe, like ModSync or SVM (ServerValueModifier)
+  - .txt and .md are usually README's or licenses.
+- Move any remaining downloaded/unzipped files to the `mods_download/remains` directory.
+
+It also keeps track of each URL downloaded in the `mods_download/mod_urls_downloaded.txt` file so it does not re-download one that has already been downloaded unless you manually remove it from or delete that file entirely.
+
+### How to use it
+This is disabled by default so first the `INSTALL_OTHER_MODS` variable needs to be set to `true`. There are two methods to specify the URLs (you can use either or both of these methods):
+- Add the URLs, one URL per line, to `mods_download/mod_urls_to_download.txt` (create the file if it's not already there).
+- Set the environment variable `MOD_URLS_TO_DOWNLOAD` to a space separated list of the URLs you want it to download.
+
+> [!WARNING] If you use both methods, it may conflict if you download multiple versions of the same mod at the same time.
+
+The URLs should point to a direct file download of a `.zip`, `.7z`, `.tar/.tar.gz`, or `.dll` file. It is assumed that compressed downloads (`.zip`, `.7z`, and `.tar/.tar.gz`) are already organized into the `BepInEx/plugins` and/or `user/mods` directory(ies). Most mod developers do this but not all of them. If a mod is not correctly organized then it will still be downloaded and extracted but the files will be moved to the `mods_download/remains` directory for you to handle manually.
+
+If a mod requires any post-installation configuration, you will still need to to this manually.
+
+A few other notes
+* You can add or change the URLs whenever you want. Any new URLs will be downloaded and installed the next time the container is restarted.
+* Removing a URL from the specified URLs does not remove the downloaded mod/files, it only stops it from checking that URL again.
+* If you want to redownload the same url, you will need to manually remove it from `mods_download/mod_urls_downloaded.txt` file.
+
+### Mod updates
+When a mod is updated, you will need to add the new URL using one of the methods above. It will be downloaded, extracted, and then merged, overwiriting any conflicting files in the installation. For simple mods that is probably enough. If the mod developer states that you will need to uninstall a previous version to update, you will have to do this manually. You may do that at any time if you want to be extra cautious.
```

Comment:
😁 
nit:

```suggestion
When a mod is updated, you will need to add the new URL using one of the methods above. It will be downloaded, extracted, and then merged, overwriting any conflicting files in the installation. For simple mods, that is probably enough. If the mod developer states that you will need to uninstall a previous version to update, you will have to do this manually. You may do that at any time if you want to be extra cautious.
```

Reply:
I'm all for correcting spelling and grammar 👍